### PR TITLE
Update strip ansi

### PR DESCRIPTION
**Pull request by alexander-akait**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [x] This is a **code refactor**
- [x] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

reduce deps, because we bundle this library + rewrite test, due bug in jest we can't use `import().then()` to load EC modules in CommonJs, so after update we will improve this

### Motivation / Use-Case

Less deps + update dep

### Breaking Changes

No

### Additional Info

No

### CODE REVIEW

File: public/other.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (anshumanv) [inline public/other.html:1]: I think extra

File: test/cli/__snapshots__/cli.test.js.snap.webpack4 [hunk 2]
```diff
@@ -1,31 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`CLI bonjour option --bonjour 1`] = `
```
Comment (anshumanv) [inline test/cli/__snapshots__/cli.test.js.snap.webpack4:3]: removed by mistake?

Comment (alexander-akait) [reply test/cli/__snapshots__/cli.test.js.snap.webpack4:3]: Yep, WIP on this

**Comment by alexander-akait** [comment]
Close in favor https://github.com/webpack/webpack-dev-server/pull/3533

**Pull request closed without merging**